classes/book.rb

```diff
@@ -0,0 +1,12 @@
+require '../item'
+class Book < Item
+  def initialize(publisher, cover_state, publish_date = Time.year)
```

Comment:
- [ ] Replace Time method
```suggestion
  def initialize(publisher, cover_state, publish_date = Time.now)
```

Reply:
publish_date is inherited from the item class so the change of publish_date has to come from there